### PR TITLE
Blog app - controllers specs

### PR DESCRIPTION
**Pull request by Innocent259**
In this project milestone I:

1. Tested the users' controller and posts' controller using Rspec.

### CODE REVIEW

File: spec/requests/users_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    it 'returns a success response' do
+      get :show, params: { id: 1 }
+      expect(response).to be_successful
+    end
+
+    it 'renders the correct template' do
+      get :show, params: { id: 1 }
+      expect(response).to render_template('show')
+    end
+  end
```
Comment (julie-ify) [inline spec/requests/users_controller_spec.rb]: - [ ] For each of these spec groups kindly add specs that ensure the response body includes the correct placeholder text.

  ```
  expect(response.body).to include("Welcome to user index page")
  ```

File: spec/requests/posts_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    it 'returns a successful response' do
+      get :show, params: { user_id: 1, id: 1 }
+      expect(response).to be_successful
+    end
+
+    it 'renders the show template' do
+      get :show, params: { user_id: 1, id: 1 }
+      expect(response).to render_template(:show)
+    end
+  end
```
Comment (julie-ify) [inline spec/requests/posts_controller_spec.rb]: - [ ] For each of these spec groups kindly add specs that ensure the response body includes the correct placeholder text. e.g

  ```
  expect(response.body).to include("Welcome to posts index page")
  ```

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @Innocent259,

Good job so far!

## Highlights!
- Good code organization and passing specs ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @Innocent259,

✔️ Well-written and passing tests

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**